client/sites-dashboard/components/sites-ellipsis-menu.tsx

```diff
@@ -127,7 +127,7 @@ const ManagePluginsItem = ( { site, recordTracks }: SitesMenuItemProps ) => {
 					has_manage_plugins_feature: hasManagePluginsFeature,
 				} )
 			}
-			info={ ! hasManagePluginsFeature && __( 'Requires a Business Plan' ) }
+			info={ ! hasManagePluginsFeature ? __( 'Requires a Business Plan' ) : undefined }
```

Comment:
The `info` prop accepts either a string or `undefined`, so a boolean was not a valid value.